### ISSUE TITLE
`textAlignment` for input modals is flipped between `center` and `right`

### ISSUE THREAD

**Issue by mafredri** [kasper/phoenix#302]
I'm testing out input modals and noticed that `textAlignment` seems to be flipped between `center` and `right`.

Example:

```js
const m = Modal.build({
	isInput: true,
	inputPlaceholder: 'Search',
	textAlignment: 'center',
});
m.show();
```

Result:

<img width="194" alt="image" src="https://user-images.githubusercontent.com/147409/167310985-83ea8d75-11b3-468e-89dd-8b3a32a67fa3.png">

I'm a bit flabbergasted as I thought this was a clear cut fix and was going to submit a PR, but I tried moving the values around in https://github.com/kasper/phoenix/blob/0028edfb3fa6728a552ab837f478c4eaee6ed16f/Phoenix/PHModalWindowController.m#L123-L125 to no avail. I swapped `@1` and `@2`, tried to re-order the items, and even swapped everything around (including left), but I couldn't get the order right.

Finally by removing one of the center options (`centre`, and having it as `@2`) I managed to get it behaving as I'd expect, but I don't think that's not the wanted solution 🤔.

- Version: 3.0.0
- macOS: 12.3.1

**Comment by kasper** [comment]
@mafredri Thanks for checking! I will try to see what’s happening here.

**Comment by kasper** [comment]
I couldn’t reproduce this. I think this is a side effect of the layout issue fixed in #324.
